youths/models.py

```diff
@@ -80,3 +81,17 @@ def __str__(self):
     def save(self, *args, **kwargs):
         self.full_clean()
         return super().save(*args, **kwargs)
+
+    serialize_fields = (
+        {"name": "birth_date", "accessor": lambda x: x.strftime("%Y-%m-%d")},
+        {"name": "school_name"},
+        {"name": "school_class"},
+        {"name": "expiration", "accessor": lambda x: x.strftime("%Y-%m-%d")},
+        {"name": "language_at_home", "accessor": lambda x: x.value},
+        {"name": "approver_first_name"},
+        {"name": "approver_last_name"},
+        {"name": "approver_phone"},
+        {"name": "approver_email"},
+        {"name": "expiration", "accessor": lambda x: x.strftime("%Y-%m-%d %H:%M")},
```

Comment:
There's two values for expiration.. is that on purpose?

Reply:
I don't think so 😃 good catch.